.circleci/config.yml

```diff
@@ -107,7 +107,9 @@ commands:
           command: |
             if [ ${CIRCLE_BRANCH} != "" ] &&
               [ ${CIRCLE_BRANCH} != "master" ] &&
-              ! git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) | grep -qvE '(.md$)|(.pdf$)'; then
+              ! git diff --name-only $(git merge-base origin/master ${CIRCLE_BRANCH}) \
+                | grep -qvE '(.md)|(.pdf)|(.png)|(.svg)|(Test.java)|(Test.kt)|(.mp4)|(.log)|(.txt)'; then
```

Comment:
I kinda want to take this logic out of this file and put it in a script that we just call here.

It's conceivable that a .txt file could influence the behavior of the product, though I don't know of any current examples of this.

Eventually I'd like to move to a model where the first job determines what future jobs to run. That is, we wouldn't see a green checkmark for halted jobs; it would just never have run at all.

Is there a way to halt with another symbol (like it does for skipped tests)? Seems like I looked at one point.